root_VS2013/programs/C#/Samples/WinStore_samples/WinStore_sample.Web/app.config

```diff
@@ -54,7 +54,10 @@
   <!-- トランザクション制御定義へのパス -->
   <add key="FxXMLTCDefinition" value="C:\root\files\resource\XML\TCDefinition.xml"/>
   <!-- 名前解決定義へのパス -->
-  <add key="FxXMLTMProtocolDefinition" value="C:\root\files\resource\XML\TMProtocolDefinition.xml"/>
+  <!-- In case of using IIS -->
+  <!--<add key="FxXMLTMProtocolDefinition" value="C:\root\files\resource\XML\TMProtocolDefinition.xml"/>-->
+  <!-- In case of using IIS Express -->
+  <add key="FxXMLTMProtocolDefinition" value="C:\root\files\resource\XML\TMProtocolDefinition2.xml"/>
```

Comment:
@daisukenishino 
@SandeepNayak077 

This file is included in WinStore_sample.Web (ASP.NET Web Services Project).
I think that this property is unnecessary.
(We do not suppose the case of calling web service from web service.)

So, I think that the modification of this pull request is unnecessary.
I close this pull request.